crates/matrix-sdk/src/account.rs

```diff
@@ -786,6 +787,40 @@ impl Account {
 
         Ok(())
     }
+
+    /// Adds the user to the account's data ignore list
+    pub(crate) async fn ignore_user(&self, user_id: &OwnedUserId) -> Result<()> {
+        let mut ignored_user_list = self.get_ignored_user_list_event_content().await?;
+        ignored_user_list.ignored_users.insert(user_id.clone(), Default::default());
+
+        // Updating the account data
+        self.set_account_data(ignored_user_list).await?;
+        // TODO: I think I should reset all the storage and perform a new local sync
+        // here but I don't know how
```

Comment:
This is the only thing I am not sure either if is needed with SS and if so, how to perform it

Reply:
okay let me know what and how I should do this if necessary because I have no idea ahahah